content/kotlin/concepts/strings/strings.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Strings'
+Description: 'A string in Kotlin is an object that holds a sequence of characters contained within a pair of double quotes (").'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+In Kotlin, a `string` is an immutable object that represents a sequence of characters.
+
+## Syntax
+
+```pseudo
+val stringName: String = "string value"
+```
+
+## Example
+
+You can also use `string` templates to include dynamic values in your strings. To do this, you enclose the dynamic value in curly braces and precede it with a dollar sign, like this:
+
+```kotlin
+val dynamicValue = 4
+val stringName: String = "The value of dynamicValue is $dynamicValue"
```

Comment:
You're using an example without the curly braces and this will lead to confusion. You should explain when the curly braces are optional and show examples with and without.

Reply:
wait I will add an example with curly braces then